plugin/bufferline.lua

```diff
@@ -0,0 +1,160 @@
+-- File: bufferline.vim
+-- Author: romgrk
+-- Description: Buffer line
+-- Date: Fri 22 May 2020 02:22:36 AM EDT
+-- !::exe [So]
+
+local bufferline = require 'bufferline'
+
+vim.opt.showtabline = 2
+
+--------------------
+-- Section: Commands
+--------------------
```

Comment:
For this section, can we extract `local create_command = vim.api.nvim_create_user_command`? Very verbose otherwise